src/index.ts

```diff
@@ -1,28 +1,33 @@
-import '/build/css/style.css'
-import { Movie_Search, Result } from './Movie'
-import { api_key } from './key.js'
+import { MovieSearch, Result } from './Movie'
+import { apiKey } from './key'
 
-const movieDisplay = document.getElementById('movie-display-container')!
+const movieDisplay = document.getElementById(
+  'movie-display-container'
+) as HTMLElement
 const movieSearchBox = document.getElementById(
   'movie-search-box'
 ) as HTMLInputElement
 
 // Get Search Term to call API
 function searchMovies(searchTerm: string): void {
   if (searchTerm.length > 0) {
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises, @typescript-eslint/no-use-before-define
     loadMovies(searchTerm)
   }
 }
 // Fetch Data, Return JSON
-const fetchMovieList = async (searchTerm: string): Promise<Movie_Search> => {
-  const API_URL = `https://api.themoviedb.org/3/search/movie?api_key=${api_key}&query=${searchTerm}`
+const fetchMovieList = async (searchTerm: string): Promise<MovieSearch> => {
+  const API_URL = `https://api.themoviedb.org/3/search/movie?api_key=${apiKey}&query=${searchTerm}`
   const movieResp = await fetch(API_URL)
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
   return movieResp.json()
 }
 
 // Movie Results
 function displayMovieResults(results: Result[]): void {
+  // eslint-disable-next-line no-restricted-syntax, @typescript-eslint/naming-convention, camelcase
```

Comment:
you can solve the camel case issue with the following:
```javascript
for (const { poster_path: posterPath } of results) {
```

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#assigning_to_new_variable_names

Reply:
I was a little hesitant to change this one because according to the data returned from the API, it only accesses the poster path with a key value of "poster_path" in the Results[]. How should I go about destructing that value?

Reply:
haha no worries. Do you want to try and push a commit to fix this, then mark this resolved?